bears/natural_language/LanguageToolBear.py

```diff
@@ -57,12 +59,18 @@ def run(self,
         natural_language = (guess_language(joined_text)
                             if natural_language == 'auto'
                             else natural_language)
-        natural_language = 'en-US' if not natural_language \
-                           else natural_language
 
-        tool = LanguageTool(natural_language, motherTongue='en_US')
-        tool.disabled.update(languagetool_disable_rules)
+        try:
+            tool = LanguageTool(natural_language, motherTongue='en_US')
+        except ValueError:
+            logging.warn(
+                "Changing the `natural_language` setting to 'en-US' as "
+                "`language_check` failed to guess a valid language'."
```

Comment:
I believe `'` was intended to be removed ;)

Reply:
(then you can use single quotes again for this line of string)

Reply:
Sorry bad mistake :worried: